### ISSUE TITLE
randomElement and named relations

### ISSUE THREAD

**Issue by pbowyer** [nelmio/alice#973]
Alice has support for picking a random item using the wildcard syntax: `@element*`. This works great when I've generated fixtures using a loop, ending up with `@element1`, `@element2` ... `@elementN`.

Where values aren't random (e.g. settings, statuses) and I manually create the fixtures, I give them descriptive names so I can easily reference them in other manually created fixtures. E.g.:
```
App\Entity\Status:
    success:
        ...
    failure: # Easier to remember than @status2
        ...
```

But in the generated fixtures, I need to pick a random status.

I have tried:
`<randomElement([@success, @failure])>`, `<randomElement(['@success', '@failure'])>`, defining a parameter that's an array of the statuses; all give the error:
```
Expected argument of type "App\Entity\Setting\Status or null", "Nelmio\Alice\Definition\Value\FixtureReferenceValue" given at property path "status".
```
Is it possible to pick a random element from a user-defined set?

**Comment by theofidry** [comment]
It should be possible, that's a proper bug that needs to be solved

**Comment by pifaace** [comment]
I confirm, I created a custom provider and a fixture like this : 
```yaml
validationGroup1:
      validationGroupId: 'validationGroup1'
      name: 'sweet validatonGroup1'
      persons: <personCollection([@johnDoe, @janeDoe])>
      orderables: '<orderableCollection([@flyer])>'
```
and the arg in my personCollection method is an array of "FixtureReferenceValue". It doesn't convert the references to object

**Comment by theofidry** [comment]
Closed by #984
